modules/standard/Version.chpl

```diff
@@ -247,11 +273,363 @@ module Version {
     the former is considered an official release and the latter a
     pre-release.
   */
-  operator sourceVersion.>=(v1: sourceVersion(?),
-                            v2: sourceVersion(?)) param : bool {
+  pragma "no doc"
+  operator versionValue.>=(v1: versionValue(?),
+                           v2: versionValue(?)) param : bool {
     param retval = spaceship(v1, v2);
     if (retval == 2) then
       compilerError("can't compare versions that only differ by commit IDs");
     return retval >= 0 && retval != 2;
   }
+
+
+  /*
+    This record represents a software version. It uses ``var`` values to
```

Comment:
I think it would be appropriate to briefly mention the relationship to `versionValue` here